docs/modoc/TokenHandler.md

```diff
@@ -152,30 +160,32 @@ func backlogFunds() : Nat
 retrieve the estimated sum of all balances in the backlog
 
 
-### Function `consolidatedFunds`
+### Function `depositedFunds`
 ``` motoko
-func consolidatedFunds() : Nat
+func depositedFunds() : Nat
 ```
 
-retrieve the sum of all successful consolidations
+retrieve the sum of all deposits. This value is nearly the same as backlogFunds(), but includes
+entries, which could not be added to backlog, for instance when balance less than fee.
+It's always >= backlogFunds()
 
 
-### Function `totalDeposit`
+### Function `creditedFunds`
 ``` motoko
-func totalDeposit() : Nat
+func creditedFunds() : Int
 ```
 
-retrieve the sum of all deposits. This value is nearly the same as backlogFunds(), but includes
-entries, which could not be added to backlog, for instance when balance less than fee
+retrieve the sum of all user credit balances.
+It can be negative because user can spend deposited funds before consolidation
 
 
-### Function `totalUsableBalance`
+### Function `usableFunds`
 ``` motoko
-func totalUsableBalance() : Nat
+func usableFunds() : Nat
 ```
 
 retrieve the sum of all user usable balances. It's tricky to cache it
-because of excluding deposits, smaller than fee, from the usable balance
+because of excluding deposits, smaller than fee, from the usable balance.
```

Comment:
The last sentence can be removed from the function description so it doesn't appear in the docs. Can you move it into the implementation where the doc generator doesn't pick it up?

Reply:
Done